package.json

```diff
@@ -45,5 +45,133 @@
     "prettier": "^2.8.6",
     "rimraf": "^3.0.2",
     "shell-quote": "^1.7.2"
+  },
+  "resolutions": {
```

Comment:
There is no way to avoid pinning all these packages?

Reply:
Ah yeah we should probably remove some of them. Lazy me just copied the list of packages from jupyterlab+lumino.

I had to pin otherwise the widgets 7 package would pull jupyterlab 3 stuff.

Reply:
Should we go further and extract the widget 7 extension into standalone packages?

Reply:
You mean outside of the repo ?

Reply:
No, just a standalone python package and being built independently to avoid dependencies conflict

Reply:
😫 this means more packaging setup, I'm honestly growing tired of our tooling and packaging.

I agree this list of pinning is annoying too we should reduce it as much as possible, let me have a look!

Reply:
Arf, it's either we pin the world, or we make a separate Python module that makes use of JupyterLab 3 (yarn 1). I hate these tools.

Reply:
Alternatively, we make a new build of ipywidgets 7 packages using JupyterLab 4. This was planned a long time ago but never done.

Reply:
Let's address this issue in a follow-up

Reply:
Agreed, let's release Voila with this.

I'm looking into building ipywidgets 7 against lab 4 as we speak so we can get rid of the pinning.